### PR TITLE
Althea

### PR DESCRIPTION
**Pull request by dontkillthesparrows**
I just want to be part of the shit

### CODE REVIEW

File: paal.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added some text here in this text file
+Hope you like it :)
```
Comment (Paalchrb) [inline paal.txt]: nice!

**Review by Paalchrb — CHANGES_REQUESTED** [review]
please dont use bad language

File: paal.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
 Added some text here in this text file
-Hope you like it :)
\ No newline at end of file
+Hope you like it :)
```
Comment (hhpettersen) [inline paal.txt:2]: DUPLICATE

Comment (dontkillthesparrows) [reply paal.txt:2]: we can be friends again

**Review by Paalchrb — APPROVED** [review]
Liker!

**Review by hhpettersen — APPROVED** [review]
LOOKS GOOD

**Pull request merged**